pkg/pool-weighted/contracts/managed/CircuitBreakerStorageLib.sol

```diff
@@ -247,75 +244,75 @@ library CircuitBreakerStorageLib {
             .insertUint(lowerBound >> _BOUND_SHIFT_BITS, _LOWER_BOUND_OFFSET, _BOUND_WIDTH)
             .insertUint(upperBound >> _BOUND_SHIFT_BITS, _UPPER_BOUND_OFFSET, _BOUND_WIDTH);
 
-        // Precompute and store the conversion ratios, used to convert percentage bounds to BPT price bounds.
-        // If the weight complement has not changed since the breaker was set, we can use the precomputed values
-        // directly, and avoid a heavy computation.
-        (uint256 lowerBoundRatio, uint256 upperBoundRatio) = CircuitBreakerLib.calcBoundaryConversionRatios(
+        // Precompute and store the adjusted bounds, used to convert percentage bounds to BPT price bounds.
+        // If the weight has not changed since the breaker was set, we can use the precomputed values directly,
+        // and avoid a heavy computation.
+        (uint256 adjustedLowerBound, uint256 adjustedUpperBound) = CircuitBreakerLib.calcAdjustedBounds(
             lowerBound,
             upperBound,
-            weightComplement
+            weight
         );
 
-        // Finally, insert these computed ratios, and return the complete set of fields.
+        // Finally, insert these computed adjusted bounds, and return the complete set of fields.
         return
             circuitBreakerState
                 .insertUint(
-                lowerBoundRatio.compress(_BOUND_RATIO_WIDTH, _MAX_BOUND_PERCENTAGE),
-                _LOWER_BOUND_RATIO_OFFSET,
-                _BOUND_RATIO_WIDTH
+                adjustedLowerBound.compress(_ADJUSTED_BOUND_WIDTH, _MAX_BOUND_PERCENTAGE),
+                _ADJUSTED_LOWER_BOUND_OFFSET,
+                _ADJUSTED_BOUND_WIDTH
             )
                 .insertUint(
-                upperBoundRatio.compress(_BOUND_RATIO_WIDTH, _MAX_BOUND_PERCENTAGE),
-                _UPPER_BOUND_RATIO_OFFSET,
-                _BOUND_RATIO_WIDTH
+                adjustedUpperBound.compress(_ADJUSTED_BOUND_WIDTH, _MAX_BOUND_PERCENTAGE),
+                _ADJUSTED_UPPER_BOUND_OFFSET,
+                _ADJUSTED_BOUND_WIDTH
             );
     }
 
     /**
-     * @notice Update the cached ratios given a new weight complement.
+     * @notice Update the cached adjusted bounds, given a new weight.
      * @dev This might be used when weights are adjusted, pre-emptively updating the cache to improve performance
      * of operations after the weight change completed. Note that this does not update the BPT price: this is still
      * relative to the last call to `setCircuitBreaker`. The intent is only to optimize the automatic bounds
      * adjustments due to changing weights.
      */
-    function updateBoundRatios(bytes32 circuitBreakerState, uint256 weightComplement) internal pure returns (bytes32) {
-        (uint256 lowerBoundRatio, uint256 upperBoundRatio) = CircuitBreakerLib.calcBoundaryConversionRatios(
+    function updateAdjustedBounds(bytes32 circuitBreakerState, uint256 currentWeight) internal pure returns (bytes32) {
```

Comment:
```suggestion
    function updateAdjustedBounds(bytes32 circuitBreakerState, uint256 newReferenceWeight) internal pure returns (bytes32) {
```

Note that this weight is not going to be the current weight as we'll pass in the weight as of **after** the weight change so it will be a future weight.